ubuntu_image/parser.py

```diff
@@ -64,6 +65,16 @@ def __call__(self, v):
             ]
 
 
+def Size32bit(v):
```

Comment:
I can do it as proposed, I just thought that using the struct module is more elegant than doing value comparison. But of course 'elegant' is of course a rather relative term. I also saw that the struct module is anyway used in ubuntu-image, so didn't see any harm in using it here as well.

I can modify it per your proposition of course, especially if you have any future ideas of using the min and max parameters for something else.